Classes/Neos/Neos/Ui/Aspects/PolicyAspect.php

```diff
@@ -52,12 +56,44 @@ class PolicyAspect
      */
     protected $nodeTypeManager;
 
+    /**
+     * @Flow\Inject
+     * @var ObjectManagerInterface
+     */
+    protected $objectManager;
+
+    /**
+     * @param ObjectManagerInterface $objectManager
+     * @return array the key is a Privilege class name; the value is "true" if privileges are configured for this class name.
+     * @Flow\CompileStatic
+     */
+    public static function getUsedPrivilegeClassNames($objectManager)
```

Comment:
Just wondering, is this method fast? Perhaps its results could be cached? Did you try to profile it?

Reply:
@dimaip Look like this method is statically cached, so should be fast ;)

Reply:
ack :) see the @Flow\CompileStatic annotation :)

Reply:
Guys, pardon my Flow ignorance, but doesn't this annotation just skips the step of Flow compilation, but would it cache its results? Or would it get re-executed for each and every node?